app/src/main/java/fr/free/nrw/commons/contributions/ContributionDao.java

```diff
@@ -155,6 +177,10 @@ public Contribution fromCursor(Cursor cursor) {
         return null;
     }
 
+    private HashMap<String, String> formatCaption(int columnIndex) {
```

Comment:
What does this exactly do ?

Reply:
I left this function empty because I was not sure about how to format captions to sore them in local database which then could be reformatted to retrieve captions. Can you please suggest what formatting would you follow?

Reply:
Shall we store them as json the exact map as it is and the same can to converted back to the map(using GSON) or the corresponding data structure (whatever it is) .?

Reply:
Like this?
```
Map<String, Object> data = new HashMap<String, Object>();
    data.put( "name", "Mars" );
    data.put( "age", 32 );
    data.put( "city", "NY" );
    JSONObject json = new JSONObject();
    json.putAll( data );
    System.out.printf( "JSON: %s", json.toString(2) );
```
output::
```
JSON: {
  "age": 32,
  "name": "Mars",
  "city": "NY"
}
```